src/ORM/DataObject.php

```diff
@@ -940,6 +949,50 @@ public function i18n_plural_name()
         return _t(static::class . '.PLURALNAME', $this->plural_name());
     }
 
+    /**
+     * Get description for this class
+     * @return null|string
+     */
+    public function classDescription()
+    {
```

Comment:
Using PHPDoc for this and `i18n_classDescription()`'s return types for BC. Since these methods already exist on `SiteTree` without strict return types, we can't add them here.